src/03-control-flow.js

```diff
@@ -14,7 +14,13 @@
  *  isEqual(10, "10");
  *  //> false
  */
-function isEqual(a, b) {}
+function isEqual(a, b) {
+  if (a === b) {
+  return true
+} else {
```

Comment:
lets indent our ` } else  { `  over one tab. and our ` return ` statements inside our blocks as well